src/roulette.py

```diff
@@ -126,3 +126,124 @@ def get(self, bin: int) -> Bin:
+            wheel.addOutcome(i, straight_bet)
+        zero_zero_bet = Outcome("00", 35)
+        bin_index_of_zero_zero_outcome = 37
+        wheel.addOutcome(bin_index_of_zero_zero_outcome, zero_zero_bet)
+
+    @staticmethod
+    def create_horizontal_split_bets(wheel: Wheel) -> None:
+        odds_for_split_bet = 17
+
+        first_column_numbers = set()
+        for i in range(12):
+            first_column_numbers.add(i * 3 + 1)
+        for i in first_column_numbers:
+            column_one_two_split_bet = Outcome(str(i) + "-" + str(i + 1), odds_for_split_bet)
+            bin_indexes_for_horizontal_split_bets = i, i + 1
+            wheel.addOutcome(bin_indexes_for_horizontal_split_bets[0], column_one_two_split_bet)
+            wheel.addOutcome(bin_indexes_for_horizontal_split_bets[1], column_one_two_split_bet)
+
+        second_column_numbers = set()
+        for i in range(12):
+            second_column_numbers.add(i * 3 + 2)
+        for i in second_column_numbers:
+            column_two_three_split_bet = Outcome(str(i) + "-" + str(i + 1), odds_for_split_bet)
+            bin_indexes_for_horizontal_split_bets = i, i + 1
+            wheel.addOutcome(bin_indexes_for_horizontal_split_bets[0],
+                             column_two_three_split_bet)
+            wheel.addOutcome(bin_indexes_for_horizontal_split_bets[1],
+                             column_two_three_split_bet)
+
+    @staticmethod
+    def create_vertical_split_bets(wheel: Wheel) -> None:
+        numbers_to_create_vertical_split_bets = set(_ for _ in range(1, 34))
+        for i in numbers_to_create_vertical_split_bets:
+            vertical_split_bet = Outcome(str(i) + "-" + str(i + 3), 17)
+            bin_indexes_for_vertical_split_outcome = i, i + 3
+            wheel.addOutcome(bin_indexes_for_vertical_split_outcome[0], vertical_split_bet)
+            wheel.addOutcome(bin_indexes_for_vertical_split_outcome[1], vertical_split_bet)
+
+    @staticmethod
+    def create_street_bets(wheel: Wheel) -> None:
+        odds_for_street_bet = 11
+        numbers_to_create_street_bets = set()
+        for i in range(12):
+            numbers_to_create_street_bets.add(i * 3 + 1)
+        for i in numbers_to_create_street_bets:
+            street_bet = Outcome(str(i) + "-" + str(i + 1) + "-" + str(i + 2), odds_for_street_bet)
+            indexes_for_street_bets = i, i + 1, i + 2
+            wheel.addOutcome(indexes_for_street_bets[0], street_bet)
+            wheel.addOutcome(indexes_for_street_bets[1], street_bet)
+            wheel.addOutcome(indexes_for_street_bets[2], street_bet)
+
+    @staticmethod
+    def create_corner_bets(wheel: Wheel) -> None:
+        odds_for_corner_bet = 8
+
+        first_column_numbers = set()
+        for i in range(11):
+            first_column_numbers.add(i * 3 + 1)
+        for i in first_column_numbers:
+            first_column_corner_bet = Outcome(str(i) + "-" + str(i + 1) + "=" + str(i + 3) +
```

Comment:
is `=` typo? should there be `-` between `(i + 3) and (i + 4)`

Reply:
Fixed. Thanks.